mllib/src/main/scala/org/apache/spark/ml/feature/StringIndexer.scala

```diff
@@ -65,12 +66,15 @@ class StringIndexer(override val uid: String) extends Estimator[StringIndexerMod
   def this() = this(Identifiable.randomUID("strIdx"))
 
   /** @group setParam */
+  def setSkipInvalid(value: Boolean): this.type = set(skipInvalid, value)
```

Comment:
Looking at the other params in the StringIndexer model as well as LogisticRegression (e.g. setThreshold) it seems that when a param makes sense to be set on the model it is exposed in both the model and the trainer.